pkg/setup/create_crd.go

```diff
@@ -41,10 +43,15 @@ func (sc Config) createBlockDeviceClaimCRD() error {
 func (sc Config) createCRD(crd *apiext.CustomResourceDefinition) error {
 	if _, err := sc.apiExtClient.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
 		if errors.IsAlreadyExists(err) {
-			// CRD is already present, no need to do anything
-			// In future can implement upgrade of CRDs here.
-			// For upgrade, a patch can be created which can then be
-			// used to upgrade the CRD
+			// if CRD already exists, we patch it with the new changes.
+			// This will also handle the upgrades of CRDs
+			patch, err := json.Marshal(crd)
+			if err != nil {
+				return fmt.Errorf("could not marshal new customResourceDefintion: %v", err)
```

Comment:
Does the error message include the name of the CRD? If not it will be good to mention which crd is having the issue.